azure/relay-sample/src/main/java/com/yanchware/fractal/azure/sample/components/Relay.java

```diff
@@ -0,0 +1,23 @@
+package com.yanchware.fractal.azure.sample.components;
+
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.AzureRegion;
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.AzureResourceGroup;
+import com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.servicebus.AzureRelay;
+
+import static com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.AzureRegion.EUROPE_WEST;
+
+public class Relay {
+  public static AzureRelay getRelay() {
+    AzureResourceGroup azureResourceGroup = AzureResourceGroup
+        .builder()
+        .withName("relay-rg")
```

Comment:
we could also use `rg-` as a prefix here. That is what we are using at customer side.